readme.md

```diff
@@ -23,6 +19,29 @@ the changes into master, and pushes those to a testing branch. When CI reports a
 successful build for this branch, master is forwarded to it. If the build fails,
 the commits never make it into master, keeping the build green at all times.
 
+
+## Using Hoff
+
+Supposing Hoff is set up to listen for the comment prefix `@hoffbot` with a
+matching GitHub user, you use it by commenting on a PR with any of the
+following commands:
+
+* `@hoffbot merge`: rebase then merge;
+* `@hoffbot merge and tag`: rebase, merge then tag.
+* `@hoffbot merge and deploy`: rebase, merge, tag then deploy;
```

Comment:
For this one, it might be good to add that hoff doesn't do the deploying. It just adds a special marker to the tag message indicating to the CI job handling it that it should also be deployed.

Reply:
Thanks for spotting this.  I've updated the readme now with the correct behaviour.